rpcs3/Emu/CMakeLists.txt

```diff
@@ -173,6 +173,16 @@ target_sources(rpcs3_emu PRIVATE
 	Cell/lv2/sys_tty.cpp
 	Cell/lv2/sys_usbd.cpp
 	Cell/lv2/sys_vm.cpp
+	Cell/lv2/sys_hid.cpp
```

Comment:
i think these should be added ordered by name, at least that's how it is usually